src/renderer.cpp

```diff
@@ -165,10 +165,7 @@ namespace big
 		for (const auto& cb : m_wndproc_callbacks)
 			cb(hwnd, msg, wparam, lparam);
 
-		if (g_gui->is_open())
-		{
-			ImGui_ImplWin32_WndProcHandler(hwnd, msg, wparam, lparam);
-		}
+		ImGui_ImplWin32_WndProcHandler(hwnd, msg, wparam, lparam);
```

Comment:
I've did precisely all of this when maybegreat48 gave me a hard on for commands. I came across an issue with inputbox glitching out and not responding. Unsure why, Pissed me off... I recommend making this not always on btw. The user can press things if any imgui window is open without knowledge. Can come back to haunt you in the future

Reply:
Yeah I discussed this with Yimura and we came to the conclusion that ImGui probably used key down events. That when I press enter on a command it will close my quick search. But the last key event ImGui received is an enter down. So when I open it again it will continue from having received this event and just start acting like a stuck key. That's our guess.

Reply:
As Flop said, this will fix a number of issues that ImGui would randomly have (stuck input fields that can't be modified, keys that remain pressed down, etc...). In our testing it did not negatively impact performance and I'd think that the ImGui team would properly optimise this function as in their implementation there's no need to disable calls to this function.

The only reason this was ever like this is because pocakking designed it that way 4 years ago.

Reply:
But it also handles mouse input. Which can close, minimize, resize and press widgets etc for a window. That was the premise of my argument..